tools/tfs-fuse.c

```diff
@@ -220,15 +220,16 @@ closure_function(2, 1, void, req_handle,
                 if ((++iov_count == IOV_MAX) || (xfer == total))
                     break;
             }
+            lseek(bound(d), offset, SEEK_SET);
```

Comment:
lseek() could be called just once (like you did for mkfs and dump), instead of at each iteration of the while loop.

Reply:
Fixed!